contrib/ros/README.md

```diff
@@ -0,0 +1,138 @@
+# rllab.contrib.ros Instructions
+## Introduction
+Most of the current environments for reinfocement learning training(e.g. gym's environments) are not particularly 
+designed for robot learning. Consequently, most of their environments are not for continuous action space task. And 
+it is also hard for robot learning researchers to use the same environment interface when they want to test their 
+algorithms on real robots. 
+
+rllab.contrib.ros's environments take a different approach. It's targeting at provide the same interfaces for both of the 
+training on real and simulation robots. It's developed by Hejia Zhang.
+## Support Platform
+| Robot        | Workstation OS | ROS    | Python | Simulation Platform | 
+|:------------:|:--------------:| :-----:|:------:|:-------------------:|
+| Sawyer       | Ubuntu16.04    | Kinetic| 3      |Gazebo 7.0           |
+## Installation
+- Assuming we got a clean system which only has rllab running.
+- Check if our robots and workstations are supported by rllab.contrib.ros    
+- Following is how we setup the rllab.contrib.ros's environment for sawyer robot
+### Setup Workstation
+#### INSTALL ROS
+##### Configure Ubuntu repositories
+Configure your Ubuntu repositories to allow "restricted," "universe," and "multiverse." **Likely, they are already configured properly, and you only need to confirm the configuration.**
+##### Setup your sources.list
+    $ sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu xenial main" > /etc/apt/sources.list.d/ros-latest.list'
+##### Setup your keys
+    $ sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
+##### Update to Latest Software Lists
+    $ sudo apt-get update
+##### Install ROS Kinetic Desktop Full
+    $ sudo apt-get install ros-kinetic-desktop-full
+##### Initialize rosdep
+rosdep enables you to easily install system dependencies for source you want to compile and is required to run some core components in ROS.
+    
+    $ sudo rosdep init
+    $ rosdep update
+##### Install rosinstall
+    $ sudo apt-get install python-rosinstall
+#### CREATE DEVELOPMENT WORKSPACE
+##### Create ROS Workspace
+    $ mkdir -p ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src
```

Comment:
You can make it easier on users by having them define <YOUR_FAVORITE_WORKSPACE_NAME> as a bash variable instead, then using it throughout the tutorial.

For example
```sh
export YOUR_ROS_WS=~/path/to/your/ros/workspace/src
```

Then, everywhere else in the tutorial use the variable, e.g.
```sh
mkdir -p ~/$YOUR_ROS_WS/src
```